attributes/nftables.rb

```diff
@@ -0,0 +1,22 @@
+default['firewall']['nftables']['defaults'][:policy] = {
+  input: 'drop',
+  forward: 'drop',
+  output: 'accept',
+}
+default['firewall']['nftables']['defaults'][:ruleset] = {
```

Comment:
Let's not inter-mix `String` and `Symbol` keys despite the fact that the `Mash` makes it work either way

Reply:
I thought this would fail cookstyle, @tas50 ?

Reply:
FYI: I ran cookstyle 7.8.0 (RuboCop 1.10.0) and it did not complain. I will use strings instead of symbols.